tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator.Console/Program.cs

```diff
@@ -24,7 +23,7 @@ public static void Main(string[] args)
 
             string sourcePath = args[0];
             AssemblyLoader.Initialize(sourcePath);
-            ExtensionsMetadataGenerator.Generate(sourcePath, args[1], s => { });
+            ExtensionsMetadataGenerator.Generate(sourcePath, args[1], s => System.Console.Error.Write(s));
```

Comment:
This issue was actually logging an exception -- but we didn't do anything with it. I verified this will not show up as an MSBuild Warning when there's an issue.